WorkoutTest/ExerciseController.swift

```diff
@@ -72,47 +81,55 @@ class ExerciseController: UIViewController {
-      myimage.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -10),
-      myimage.heightAnchor.constraint(equalToConstant: 350),
+      myScrollView.topAnchor.constraint(equalTo: view.topAnchor),
+      myScrollView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+      myScrollView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+      myScrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+      
+      myImage.topAnchor.constraint(equalTo: myScrollView.contentLayoutGuide.topAnchor, constant: 10),
+      myImage.leadingAnchor.constraint(equalTo: myScrollView.leadingAnchor, constant: 10),
+      myImage.trailingAnchor.constraint(equalTo: myScrollView.frameLayoutGuide.trailingAnchor, constant: -10),
+      myImage.heightAnchor.constraint(equalToConstant: 350),
       
-      nameExercise.topAnchor.constraint(equalTo: myimage.bottomAnchor, constant: 10),
-      nameExercise.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 10),
-      nameExercise.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -10),
+      nameExercise.topAnchor.constraint(equalTo: myImage.bottomAnchor, constant: 10),
+      nameExercise.leadingAnchor.constraint(equalTo: myScrollView.leadingAnchor, constant: 10),
+      nameExercise.trailingAnchor.constraint(equalTo: myScrollView.frameLayoutGuide.trailingAnchor, constant: -10),
       
       
       descriptionLabel.topAnchor.constraint(equalTo: nameExercise.bottomAnchor, constant: 10),
-      descriptionLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 10),
-      descriptionLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -10),
+      descriptionLabel.leadingAnchor.constraint(equalTo: myScrollView.leadingAnchor, constant: 10),
+      descriptionLabel.trailingAnchor.constraint(equalTo: myScrollView.frameLayoutGuide.trailingAnchor, constant: -10),
+      descriptionLabel.bottomAnchor.constraint(equalTo: myScrollView.contentLayoutGuide.bottomAnchor),
       
-      backButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 10),
+      backButton.leadingAnchor.constraint(equalTo: myScrollView.leadingAnchor, constant: 10),
       backButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -10),
       backButton.widthAnchor.constraint(equalToConstant: 100),
       backButton.heightAnchor.constraint(equalToConstant: 50),
       
       nextButton.leadingAnchor.constraint(equalTo: backButton.trailingAnchor, constant: 10),
       nextButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -10),
-      nextButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -10),
+      nextButton.trailingAnchor.constraint(equalTo: myScrollView.frameLayoutGuide.trailingAnchor, constant: -10),
       nextButton.heightAnchor.constraint(equalToConstant: 50),
+      
```

Comment:
should not be inside scroll view. Should be a subview of a self.view.  Update constraints accordingly